tests/integration_tests.rs

```diff
@@ -674,6 +674,38 @@ async fn txn_lock_keys() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+#[serial]
+async fn txn_lock_keys_error_handle() -> Result<()> {
+    init().await?;
+    let client = TransactionClient::new_with_config(pd_addrs(), Default::default(), None).await?;
+    let mut rng = thread_rng();
+
+    let k = gen_u32_keys(4, &mut rng)
```

Comment:
In this test I think it would be better not to use rand numbers. We can manually choose 4 keys that should locate in different regions.